BUG: Could not resolve "../presentation/app" when yarn test

```
X [ERROR] TS2307: Cannot find module '../presentation/app' or its corresponding type declarations. [plugin angular-compiler]

    src/app/presentation/app.spec.ts:3:20:
      3 │ import { App } from '../presentation/app';
        ╵                     ~~~~~~~~~~~~~~~~~~~~~


X [ERROR] Could not resolve "../presentation/app"

    src/app/presentation/app.spec.ts:3:20:
      3 │ import { App } from '../presentation/app';
```

--- a/apps/web/questlog-web/src/app/app.config.ts
+++ b/apps/web/questlog-web/src/app/app.config.ts
@@ -1,7 +1,7 @@
 import { ApplicationConfig, provideBrowserGlobalErrorListeners, provideZonelessChangeDetection } from '@angular/core';
 import { provideRouter } from '@angular/router';
 
-import { routes } from '../app.routes';
+import { routes } from './app.routes';
 
 export const appConfig: ApplicationConfig = {
   providers: [

--- a/apps/web/questlog-web/src/app/presentation/app.ts
+++ b/apps/web/questlog-web/src/app/presentation/app.ts
@@ -1,12 +1,12 @@
 import { Component, signal, inject, OnInit } from '@angular/core';
 import { RouterModule, RouterOutlet } from '@angular/router';
-import { AppStore } from './application/app.store';
+import { AppStore } from '../application/app.store';
 
 @Component({
   selector: 'app-root',
   imports: [RouterModule, RouterOutlet],
-  templateUrl: './presentation/app.html',
-  styleUrl: './presentation/app.scss'
+  templateUrl: 'app.html',
+  styleUrl: 'app.scss'
 })
 
 export class App implements OnInit{

--- a/apps/web/questlog-web/src/main.ts
+++ b/apps/web/questlog-web/src/main.ts
@@ -1,6 +1,6 @@
 import { bootstrapApplication } from '@angular/platform-browser';
-import { appConfig } from './app/presentation/app.config';
-import { App } from './app/app';
+import { appConfig } from './app/app.config';
+import { App } from './app/presentation/app';
 import 'bootstrap/dist/js/bootstrap.bundle.min.js';
 
 bootstrapApplication(App, appConfig)

